gems/sidekiq/7.0/middleware.rbs

```diff
@@ -59,3 +59,5 @@ module Sidekiq
     def redis: () { (RedisClientAdapter::CompatClient) -> void } -> void
   end
 end
+
+module Sidekiq::ClientMiddleware = Sidekiq::ServerMiddleware
```

Comment:
Note that I tried the following code first, but it didn't work.


```rbs
module Sidekiq
  # ...


  module ClientMiddleware = ServerMiddleware
end
```

With this RBS, `include Sidekiq::ClientMiddleware` can't find this module.

Probably it's an RBS bug. I'll investigate and report it.

Reply:
reported. https://github.com/ruby/rbs/issues/2293